tldw_Server_API/app/api/v1/endpoints/audio/audio_transcriptions.py

```
@@ -1121,11 +1217,34 @@ def _fmt_vtt_timestamp(total_seconds: float) -> str:
+    except HTTPException as exc:
+        detail = exc.detail if isinstance(exc.detail, dict) else {}
+        detail_status = str(
+            detail.get("status")
+            or detail.get("detail_status")
+            or ""
+        ).strip().lower()
+        if exc.status_code == status.HTTP_429_TOO_MANY_REQUESTS:
+            _emit_error_metrics(status_label="quota_exceeded", reason="quota")
+        elif exc.status_code in {
+            status.HTTP_400_BAD_REQUEST,
+            HTTP_413_TOO_LARGE,
```

Ruff (0.15.9)

[error] Undefined name `HTTP_413_TOO_LARGE`

(F821)